core/handlers/v2/simulation_views_v6.go

```diff
@@ -87,3 +111,11 @@ func (this ResponseDetailsViewV6) GetHeaders() map[string][]string { return this
 
 // Gets FixedDelay - required for interfaces.Response
 func (this ResponseDetailsViewV6) GetFixedDelay() int { return this.FixedDelay }
+
+// Gets LogNormalDelay - required for interfaces.Response
+func (this ResponseDetailsViewV6) GetLogNormalDelay() interfaces.ResponseDelay {
+	return this.LogNormalDelay
```

Comment:
a bit overkilled to add `HasLogNormalDelay` to the interface just for checking the delay is nil. You can try this, it returns an actual `nil` 
```
if this.LogNormalDelay != nil {
       return this.LogNormalDelay
}
return nil
```

Reply:
wow, nice trick :smile: